### PR TITLE
📖 improve release note generator

### PR DESCRIPTION
**Pull request by tuminoid**
Manual cherry pick of #1177.

### CODE REVIEW

**Comment by tuminoid** [comment]
/test-ubuntu-e2e-integration-release-1-5
Flake

File: hack/tools/release/notes.go [hunk 1]
```diff
@@ -186,9 +192,9 @@ func run() int {
 	}
 
 	fmt.Printf("The image for this release is: %v\n", latestTag)
-	fmt.Printf("Ironic release is capm3-%v\n", latestTag)
-	fmt.Printf("Mariadb release is capm3-%v\n", latestTag)
-	fmt.Println("_Thanks to all our contributors!_ 😊")
+	fmt.Printf("Ironic image tag is capm3-%v\n", latestTag)
+	fmt.Printf("Mariadb image tag is capm3-%\nv", latestTag)
```
Comment (lentzi90) [inline hack/tools/release/notes.go:196]: I guess we should fix this here since it is anyway a manual cherry pick?

Comment (tuminoid) [reply hack/tools/release/notes.go:196]: Good catch, no idea how that did not get fixed

**Review by lentzi90 — COMMENTED** [review]
/approve

**Review by Sunnatillo — APPROVED** [review]
/lgtm

**Pull request merged**